lib/libzstd/Makefile.am

```diff
@@ -0,0 +1,44 @@
+include $(top_srcdir)/config/Rules.am
+
+VPATH = \
+	$(top_srcdir)/contrib/zstd \
+	$(top_srcdir)/contrib/zstd/common \
+	$(top_srcdir)/contrib/zstd/compress \
+	$(top_srcdir)/contrib/zstd/decompress \
+	$(top_srcdir)/module/zstd
+
+# Quiet warnings about frame size due to use of an unmodified zstd lib
+AM_CFLAGS += -Wframe-larger-than=18000
```

Comment:
Same: maybe turn this into:
`# Quiet warnings about frame size due to unused code in unmodified zstd lib`